controllers/replication.storage/volumereplication_controller.go

```diff
@@ -421,7 +423,10 @@ func getScheduleTime(parameters map[string]string, logger logr.Logger) time.Dura
 		logger.Error(err, "failed to parse time: %v", rawScheduleTime)
 		return defaultScheduleTime
 	}
-	return scheduleTime
+	// Reduce the schedule time by half to get the latest update and also to
+	// avoid the inconsistency between the last sync time in the VR and the
+	// Storage system.
+	return scheduleTime / 2
```

Comment:
(nit) Protect with a lower bound of 1 minute? IOW if schedule is less than 2 minutes, then let's not reconcile that fast? Typically I expect the replication schedule to be around 5 minutes at the lower bound.